lfs/client.go

```diff
@@ -156,12 +155,13 @@ func DownloadCheck(oid string) (*objectResource, *WrappedError) {
 }
 
 func DownloadObject(obj *objectResource) (io.ReadCloser, int64, *WrappedError) {
-	req, creds, err := obj.NewRequest("download", "GET")
+	tracerx.Printf("WTF: %v", obj)
```

Comment:
Gah, 2nd debug message that I left in here.

*force pushes*

These are not the commits you are looking for.